app/players/elves/elf.py

```diff
@@ -0,0 +1,12 @@
+from app.players.player import Player
+from abc import ABC, abstractmethod
+
+
+class Elf(Player, ABC):
+    def __init__(self, musical_instrument: int) -> str:
+        self.musical_instrument = musical_instrument
```

Comment:
![image](https://github.com/mate-academy/py-elves-and-dwarves/assets/58142395/b0f43c03-c82e-409b-8c7f-8b28aa32afcc)